rotkehlchen/chain/ethereum/modules/uniswap/utils.py

```diff
@@ -253,3 +268,220 @@ def historical_uniswap_v2_lp_price(
+                ],
+            )
+            slots_0 = [
+                entry[0].decode(entry[1][1], 'slot0')
+                for entry in zip(pool_contracts, slots_0_multicall)
+            ]
+            tokens_a = [
+                ethereum.get_basic_contract_info(to_checksum_address(position[2]))
+                for position in positions
+            ]
+            tokens_b = [
+                ethereum.get_basic_contract_info(to_checksum_address(position[3]))
+                for position in positions
+            ]
+            price_ranges = [
+                calculate_price_range(
+                    tick_lower=entry[2][5],
+                    tick_upper=entry[2][6],
+                    decimal_a=entry[0]['decimals'],
+                    decimal_b=entry[1]['decimals'],
+                )
+                for entry in zip(tokens_a, tokens_b, positions)
+            ]
+            amounts_a = [
+                calculate_amount_a(
+                    tick=entry[0][1],
+                    tick_bottom=entry[1][5],
+                    tick_top=entry[1][6],
+                    liquidity=entry[1][7],
+                    decimals=entry[2]['decimals'],
+                )
+                for entry in zip(slots_0, positions, tokens_a)
+            ]
+            amounts_b = [
+                calculate_amount_b(
+                    tick=entry[0][1],
+                    tick_bottom=entry[1][5],
+                    tick_top=entry[1][6],
+                    liquidity=entry[1][7],
+                    decimals=entry[2]['decimals'],
+                )
+                for entry in zip(slots_0, positions, tokens_b)
+            ]
+            for entry in zip(tokens_ids, pool_addresses, positions, price_ranges, tokens_a, tokens_b, amounts_a, amounts_b):  # noqa: 501
+                if entry[6] > ZERO or entry[7] > ZERO:
+                    entry[4].update({'amount': entry[6], 'address': entry[2][2]})
+                    entry[5].update({'amount': entry[7], 'address': entry[2][3]})
+                    balances.append(_decode_v3_result(userdb, entry, known_assets, unknown_assets))
+    return balances
+
+
+def compute_pool_address(
+    factory_address: ChecksumEthAddress,
+    token0_address: ChecksumEthAddress,
+    token1_address: ChecksumEthAddress,
+    fee: int,
+) -> ChecksumAddress:
+    """
+    Generate the pool address from the Uniswap Factory Address, pair of tokens
+    and the fee using CREATE2 opcode.
+    """
+    token_0 = to_checksum_address(token0_address)
+    token_1 = to_checksum_address(token1_address)
+    factory_address = to_checksum_address(factory_address)
+    parameters = []
+    # Sort the addresses
+    if int(token_0, 16) < int(token_1, 16):
+        parameters = [token_0, token_1, fee]
+    else:
+        parameters = [token_1, token_0, fee]
+    abi_encoded_1 = encode_abi(
+        ['address', 'address', 'uint24'],
+        parameters,
+    )
+    # pylint: disable=no-value-for-parameter
```

Comment:
Why are this comments here?

Reply:
Pylint raises an error for the line when `make lint` is run.
Something about it not recognising that it's a classmethod.